src/groce_cli.py

```
@@ -0,0 +1,112 @@
+
+async def handle_cli_message(message: discord.Message) -> None:
+    """Handle a message in a groce-cli channel."""
+    print("Handling groce-cli message:", message.content)
+    commands = get_commands()
+    print("Available commands:", commands.keys())
+    user_cmd = message.content.split()[0].lower()
+    command = commands.get(user_cmd)
+    if command:
+        try:
+            await command(message)
+        except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)